### PR TITLE
feat: swcr scaling improvements

### PR DESCRIPTION
**Pull request by Gustav-Eikaas**
Updated swcr sidesheet 
Also made some styled-components changes from the v6 update, all propes should be prefixed with $.

### CODE REVIEW

File: libs/swcrsidesheet/src/lib/ui-sidesheet/SwcrSidesheet.tsx [hunk 1]
```diff
@@ -25,43 +30,87 @@ type SwcrProps = {
...
+        </ErrorWrapper>
+      </div>
+    );
+  }
+
+  return <SwcrSidesheetComponent id={id} item={data} close={closeSidesheet} />;
+};
+
+export const SwcrSidesheetComponent = ({ id, close: closeSidesheet, item }: Required<SwcrProps>) => {
+  console.log(item);
```
Comment (kjellhaaland) [inline libs/swcrsidesheet/src/lib/ui-sidesheet/SwcrSidesheet.tsx:74]: Remove the console.log from here

File: libs/swcrsidesheet/src/lib/ui-sidesheet/SwcrSidesheet.tsx [hunk 2]
```diff
@@ -25,43 +30,87 @@ type SwcrProps = {
...
+
   const { data: signatures, isLoading: signaturesFetching, error } = useSignatures(id);
   const [activeTab, setActiveTab] = useState(0);
   const handleChange = (index: number) => {
     setActiveTab(index);
   };
-  const attachmentsUrls = item?.swcrUrl.replace('#', '#tab=attachments&');
+  //TODO: move to backend
+  // const attachmentsUrls = item?.swcrUrl.replace('#', '#tab=attachments&');
```
Comment (kjellhaaland) [inline libs/swcrsidesheet/src/lib/ui-sidesheet/SwcrSidesheet.tsx:82]: This seems to be outdated? Should be removed.

**Review by kjellhaaland — CHANGES_REQUESTED** [review]
Added some comments

**Comment by Gustav-Eikaas** [comment]
Fixed!

**Review by kjellhaaland — APPROVED** [review]

**Pull request merged**